.rubocop.yml

```diff
@@ -12,3 +12,10 @@ Naming/VariableNumber:
 Style/FormatStringToken:
   Exclude:
     - 'config/routes.rb'
+
+Style/MethodMissing:
+  Exclude:
+    - 'app/helpers/admin/edition_routes_helper.rb'
+    - 'app/presenters/edition_collection_presenter.rb'
+    - 'features/support/retrospective_stub.rb'
+    - 'lib/localised_model.rb'
```

Comment:
Might be useful having a comment on this commit, or bit of the config to say why we've disabled this, particularly if there's some work we could do later to remove these exclusions.

Reply:
👍 Additional information added to the commit message.